google/cloud/firestore_v1/bulk_writer.py

```diff
@@ -155,15 +226,15 @@ class BulkWriter(AsyncBulkWriterMixin):
         db = firestore.Client()
         bulk_writer = db.bulk_writer()
 
-        # Attach an optional success listener. This will be called once per
-        # document.
+        # Attach an optional success listener to be called once per document.
         bulk_writer.on_write_result(
-            lambda reference, result: print(f'Saved {reference._document_path}')
+            lambda reference, result, bw: print(f'Saved {reference._document_path}')
```

Comment:
just verifying this is intentional?

Reply:
Yep, that new parameter is real; though I've renamed it to `bulk_writer`.